jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/AbstractAdvanceRuntimeDataServiceImpl.java

```diff
@@ -190,16 +212,28 @@ public void setEmf(EntityManagerFactory emf) {
                                  ") TABLE_PROC_VAR ON TABLE_PROC_VAR.processInstanceId = pil.processInstanceId \n");
         }
 
-        if (!owners.isEmpty()) {
-            derivedTables.append("INNER JOIN ( \n" +
-                             "           SELECT DISTINCT po.task_id \n" +
-                             "           FROM PeopleAssignments_PotOwners po \n" +
-                             "           WHERE po.entity_id IN (:owners) \n" +
-                             "           GROUP BY po.task_id \n" +
-                             "           HAVING COUNT(po.entity_id) = :num_owners \n" +
-                                 ") pot ON pot.task_id = task.id ");
+        Boolean exclusiveQuery = isExclusiveQuery(ownersArg);
+        if (ownersArg != null) {
+            if (exclusiveQuery) {
+                derivedTables.append("INNER JOIN ( \n" +
+                        "           SELECT DISTINCT po.task_id \n" +
+                        "           FROM PeopleAssignments_PotOwners po \n" +
+                        "           WHERE po.entity_id IN (:owners) \n" +
+                        "           GROUP BY po.task_id \n" +
+                        "           HAVING COUNT(po.entity_id) = :num_owners \n" +
+                        ") pot ON pot.task_id = task.id ");
+            } else {
+                derivedTables.append("INNER JOIN ( \n" +
+                        "           SELECT DISTINCT po.task_id \n" +
+                        "           FROM PeopleAssignments_PotOwners po \n" +
+                        "           WHERE po.entity_id IN (:owners) \n" +
+                        "           GROUP BY po.task_id \n" +
```

Comment:
group by is not required with the any operator.

Reply:
done